tests/NPSP.py

```diff
@@ -672,11 +710,36 @@ def select_value_from_list(self,list_name,value):
         self.selenium.set_focus_to_element(locator)       
         self.selenium.select_from_list_by_label(loc,value) 
         
+    def select_value_from_bge_dd(self,list_name,value):
+        if list_name == 'Method': 
+            locator = npsp_lex_locators['bge']['dd'].format(list_name)
+            loc = self.selenium.get_webelement(locator)
+            self.selenium.set_focus_to_element(locator)       
+            self.selenium.select_from_list_by_label(loc,value) 
+        else:
+            locator = npsp_lex_locators['bge']['list'].format(list_name)
+            loc = self.selenium.get_webelement(locator)
+            self.selenium.set_focus_to_element(locator)       
+            self.selenium.select_from_list_by_label(loc,value)  
+        
     def select_multiple_values_from_list(self,list_name,*args): 
         locator = npsp_lex_locators['npsp_settings']['multi_list'].format(list_name)
         loc = self.selenium.get_webelement(locator)
         self.selenium.set_focus_to_element(locator)       
-        self.selenium.select_from_list_by_label(loc,*args)      
+        self.selenium.select_from_list_by_label(loc,*args) 
+        
+    def select_duellist_values(self,object,list_name, *args):
+        """ """
+        if object == 'Donation Matching Rule':
```

Comment:
`object` is a built-in name in Python so it's best to avoid replacing it with your own variables. Most code I've seen uses `obj` instead.

Reply:
updated it